<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRowDialog</class>
 <widget class="QDialog" name="AddRowDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="IDLabel">
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="IDLineEdit">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="PClassLabel">
     <property name="text">
      <string>PClass</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBox">
     <property name="maxVisibleItems">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="survivedLabel">
     <property name="text">
      <string>Survived</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="nameLineEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="sexLabel">
     <property name="text">
      <string>Sex</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="sexLineEdit"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="ageLabel">
     <property name="text">
      <string>Age</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="spinBox">
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
